_sources/GeneralIntro/ThePythonProgrammingLanguage.rst

```diff
@@ -61,74 +61,14 @@ level language, called **byte code**, and then interpreted by a program called
 a **virtual machine**. Python uses both processes, but because of the way
 programmers interact with it, it is usually considered an interpreted language.
 
-In this book, and for the first half of this course, you will not need to install
+For the core material in this book, you will not need to install
 or run python natively on your computer. Instead, you'll be writing simple
-programs and executing them right in your browser. About midway through
-the course, you will install python, and then the next few paragraphs will
-apply. Consider them a preview for now. 
+programs and executing them right in your browser.
 
-There are two ways to use the Python interpreter: *shell mode* and *program
```

Comment:
Why are we deleting all of this?

Reply:
I'm trying to pare down stuff that I don't think is needed to get started. No one needs this at this point in the textbook. Some students (those who will shift to jupyter notebooks rather than interpreter and .py files) won't need this ever.